include/deal.II/base/index_set.h

```diff
@@ -975,7 +982,7 @@ class IndexSet
  * @relatesalso IndexSet
  */
 inline IndexSet
-complete_index_set(const unsigned int N)
+complete_index_set(const types::global_dof_index N)
```

Comment:
The comment above applies here as well.

Reply:
No, this one is definitely wrong since it just passes `N` to the constructor of `IndexSet`. It must be the same type. In fact, I would write it as `IndexSet::size_type` to be really clear.

Reply:
Good point. Fixed.